import_pg_dump.sh

```diff
@@ -16,7 +17,22 @@ if [[ ! -f "$DUMPFILE" ]]; then
     exit 1
 fi
 
-dropdb dryad_repo
-createdb -U dryad_app -E UNICODE dryad_repo
-pg_restore -j $THREADS -d dryad_repo -U dryad_app $DUMPFILE 
+if [[ "$DBHOST" == "" ]]; then
+	DBHOST=$DRYAD_DB_HOST
+    echo "Exiting."
```

Comment:
Why exit here? The next "if" will never be used, because the script always exits here when DBHOST is absent.